From d92dfab842cb72f695b704dfc8bf889378864432 Mon Sep 17 00:00:00 2001
From: build <build@obs.com>
Date: Tue, 26 Jan 2021 15:43:01 +0800
Subject: [PATCH] isula-build:remove docker related path for authentication

reason: remove docker related authentication path ${HOME}/.docker/config.json and ${HOME}/.dockercfg
---
 .../containers/image/v5/pkg/docker/config/config.go        | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/vendor/github.com/containers/image/v5/pkg/docker/config/config.go b/vendor/github.com/containers/image/v5/pkg/docker/config/config.go
index cf82ee8..1165d15 100644
--- a/vendor/github.com/containers/image/v5/pkg/docker/config/config.go
+++ b/vendor/github.com/containers/image/v5/pkg/docker/config/config.go
@@ -219,14 +219,7 @@ func getAuthFilePaths(sys *types.SystemContext, homeDir string) []authPath {
 		paths = append(paths,
 			authPath{path: filepath.Join(dockerConfig, "config.json"), legacyFormat: false},
 		)
-	} else {
-		paths = append(paths,
-			authPath{path: filepath.Join(homeDir, dockerHomePath), legacyFormat: false},
-		)
 	}
-	paths = append(paths,
-		authPath{path: filepath.Join(homeDir, dockerLegacyHomePath), legacyFormat: true},
-	)
 	return paths
 }
 
-- 
2.27.0

